<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CollectStandardizationWidget</class>
 <widget class="QWidget" name="CollectStandardizationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pB_StartCollectPhoto">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始采图</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_TxtName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>zoom109robot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_CalculateZoomValue">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>计算新表</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>输入文件名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_CutZoomTxt">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>切换zoom表</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_DefaultsTxt">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>使用默认zoom表</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
